src/components/SearchBar/SearchBar.tsx

```diff
@@ -16,14 +21,29 @@ const SearchBar = (props: Props) => {
   const dispatch: DispatchType = useDispatch();
 
   const [searchValue, setSearchValue] = useState<string>("");
+  const [searchOption, setSearchOption] = useState<string>("1");
 
   useEffect(() => {
     dispatch(getAllJobsApi());
   }, [dispatch]);
 
   const handleSearchJob = () => {
+    if (!searchValue) {
+      Swal.fire({
+        icon: "error",
+        title: "Oops...",
+        text: "You must type something before search",
+      });
+      return;
+    }
     const tempValue = searchValue.trim().toLowerCase();
-    dispatch(searchJobApi(tempValue));
+    if (searchOption === "1") {
```

Comment:
Please use switch case instead of use "if/else"

Reply:
Fixed